### PR TITLE
Explain --pod-selector

### PR DESCRIPTION
**Pull request by fbac**
Add a bit of information on how --pod-selector works for a less confusing experince with the oc command.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@mburke5678, thoughts?

**Comment by mburke5678** [comment]
@sjenning Is this something you can comment on?

**Comment by mburke5678** [comment]
@sjenning Is this something you can comment on?

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -316,6 +316,8 @@ xref:../architecture/core_concepts/deployments.adoc#replication-controllers[repl
 controller] can be evacuated; the replication controllers create new pods on
 other nodes and remove the existing pods from the specified node(s). Bare pods,
 meaning those not backed by a replication controller, are unaffected by default.
+You can evacuate a subset of nodes by specifying a pod-selector. Pod selector is
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply admin_guide/manage_nodes.adoc]: +1

[the remark this answers is not in the document]
Comment (fbac) [reply admin_guide/manage_nodes.adoc]: Fixed typo

**Pull request merged**